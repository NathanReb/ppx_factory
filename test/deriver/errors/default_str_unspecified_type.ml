type 'a t = int * 'a
[@@deriving default]
