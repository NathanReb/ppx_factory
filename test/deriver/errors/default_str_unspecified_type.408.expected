File "default_str_unspecified_type.ml", line 1, characters 18-20:
1 | type 'a t = int * 'a
                      ^^
Error: ppx_factory: can't derive default for unspecified type
